lib/__tests__/Autocomplete-test.js

```diff
@@ -88,7 +96,7 @@ describe('Autocomplete kewDown->ArrowDown event handlers', () => {
 
   it('should highlight the "n+1" item in the menu when "n" is selected', () => {
     autocompleteWrapper.setState({'isOpen': true});
-    
```

Comment:
There is a lot of noise in your last commit. Please revert and ensure you're only touching code that relates to this PR. Whitespace cleanup can be done in a separate PR.